### ISSUE TITLE
r-stringi icu 58.* version mismatch seem to be back with conda-forge R 3.5.1 on Linux

### ISSUE THREAD

**Issue by hermidalc** [conda-forge/r-stringi-feedstock#18]
Latest conda-forge `r-stringi` fails to load with `libicui18n.so.58: cannot open shared object file: No such file or directory` error.

I believe only a few days ago it was working and I did a `conda update --all` some time and then it broke.

```R
> library(stringi)
Error: package or namespace load failed for ‘stringi’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/hermidalc/soft/miniconda3/lib/R/library/stringi/libs/stringi.so':
  libicui18n.so.58: cannot open shared object file: No such file or directory
```

Steps to reproduce on a fresh Miniconda3 install:

```bash
conda config --add channels conda-forge
conda update --all
conda install r-base=3.5.1 r-stringi
R
> library(stringi)
Error: package or namespace load failed for ‘stringi’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/hermidalc/soft/miniconda3/lib/R/library/stringi/libs/stringi.so':
  libicui18n.so.58: cannot open shared object file: No such file or directory
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/hermidalc/soft/miniconda3:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_r-mutex                  1.0.1               anacondar_1    conda-forge
asn1crypto                0.24.0                py37_1003    conda-forge
binutils_impl_linux-64    2.31.1               h6176602_1  
binutils_linux-64         2.31.1               h6176602_7  
bwidget                   1.9.11                        0    conda-forge
bzip2                     1.0.8                h516909a_0    conda-forge
ca-certificates           2019.6.16            hecc5488_0    conda-forge
cairo                     1.16.0            hfb77d84_1002    conda-forge
certifi                   2019.6.16                py37_1    conda-forge
cffi                      1.12.3           py37h8022711_0    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
conda                     4.7.10                   py37_0    conda-forge
conda-package-handling    1.3.11                   py37_0    conda-forge
cryptography              2.7              py37h72c5cf5_0    conda-forge
curl                      7.65.3               hf8cf82a_0    conda-forge
fontconfig                2.13.1            h86ecdb6_1001    conda-forge
freetype                  2.10.0               he983fc9_0    conda-forge
fribidi                   1.0.5             h516909a_1002    conda-forge
gcc_impl_linux-64         7.3.0                habb00fd_1    conda-forge
gcc_linux-64              7.3.0                h553295d_7    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
gfortran_impl_linux-64    7.3.0                hdf63c60_1  
gfortran_linux-64         7.3.0                h553295d_7  
glib                      2.58.3            h6f030ca_1002    conda-forge
graphite2                 1.3.13            hf484d3e_1000    conda-forge
gsl                       2.5                  h294904e_0    conda-forge
gxx_impl_linux-64         7.3.0                hdf63c60_1    conda-forge
gxx_linux-64              7.3.0                h553295d_7    conda-forge
harfbuzz                  2.4.0                h9f30f68_2    conda-forge
icu                       64.2                 he1b5a44_0    conda-forge
idna                      2.8                   py37_1000    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
krb5                      1.16.3            h05b26f9_1001    conda-forge
libarchive                3.3.3             hb44662c_1005    conda-forge
libblas                   3.8.0               10_openblas    conda-forge
libcblas                  3.8.0               10_openblas    conda-forge
libcurl                   7.65.3               hda55be3_0    conda-forge
libedit                   3.1.20170329      hf8c457e_1001    conda-forge
libffi                    3.2.1             he1b5a44_1006    conda-forge
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_0  
libiconv                  1.15              h516909a_1005    conda-forge
libopenblas               0.3.6                h6e990d7_5    conda-forge
libpng                    1.6.37               hed695b0_0    conda-forge
libssh2                   1.8.2                h22169c7_2    conda-forge
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.0.10            h57b8799_1003    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.9                hee79883_2    conda-forge
lz4-c                     1.8.3             he1b5a44_1001    conda-forge
lzo                       2.10              h14c3975_1000    conda-forge
make                      4.2.1             h14c3975_2004    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
openblas                  0.3.6                h6e990d7_5    conda-forge
openssl                   1.1.1c               h516909a_0    conda-forge
pango                     1.42.4               he7ab937_0    conda-forge
pcre                      8.41              hf484d3e_1003    conda-forge
pip                       19.2.1                   py37_0    conda-forge
pixman                    0.38.0            h516909a_1003    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
pycosat                   0.6.3           py37h14c3975_1001    conda-forge
pycparser                 2.19                     py37_1    conda-forge
pyopenssl                 19.0.0                   py37_0    conda-forge
pysocks                   1.7.0                    py37_0    conda-forge
python                    3.7.3                h33d41f4_1    conda-forge
python-libarchive-c       2.8                   py37_1004    conda-forge
r-base                    3.5.1             hfb2a302_1009    conda-forge
r-stringi                 1.4.3             r35h0357c0b_2    conda-forge
readline                  8.0                  hf8c457e_0    conda-forge
requests                  2.22.0                   py37_1    conda-forge
ruamel_yaml               0.15.71         py37h14c3975_1000    conda-forge
setuptools                41.0.1                   py37_0    conda-forge
six                       1.12.0                py37_1000    conda-forge
sqlite                    3.29.0               hcee41ef_0    conda-forge
tk                        8.6.9             hed695b0_1002    conda-forge
tktable                   2.10                 h555a92e_1    conda-forge
tqdm                      4.32.2                     py_0    conda-forge
urllib3                   1.25.3                   py37_0    conda-forge
wheel                     0.33.4                   py37_0    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.10               h516909a_0    conda-forge
xorg-libsm                1.2.3             h84519dc_1000    conda-forge
xorg-libx11               1.6.8                h516909a_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
yaml                      0.1.7             h14c3975_1001    conda-forge
zlib                      1.2.11            h516909a_1005    conda-forge
zstd                      1.4.0                h3b9ef0a_0    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : base
    active env location : /home/hermidalc/soft/miniconda3
            shell level : 1
       user config file : /home/hermidalc/.condarc
 populated config files : /home/hermidalc/.condarc
          conda version : 4.7.10
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages : 
       base environment : /home/hermidalc/soft/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/hermidalc/soft/miniconda3/pkgs
                          /home/hermidalc/.conda/pkgs
       envs directories : /home/hermidalc/soft/miniconda3/envs
                          /home/hermidalc/.conda/envs
               platform : linux-64
             user-agent : conda/4.7.10 requests/2.22.0 CPython/3.7.3 Linux/5.1.18-300.fc30.x86_64 fedora/30 glibc/2.29
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```
</details>

**Comment by hermidalc** [comment]
I think I found the problem, the latest `conda-forge` r-base 3.5.1 build 1009 released yesterday depends on `icu >=64.2,<65.0a0` vs the previous build 1007 from June `icu >=58.2,<59.0a0`.  So I guess latest `r-stringi` for R 3.5.1 needs to be rebuilt for `icu 64.*`.

```bash
$ conda search --info conda-forge::r-base=3.5.1
...
r-base 3.5.1 hea2b9be_1007
--------------------------
file name   : r-base-3.5.1-hea2b9be_1007.tar.bz2
name        : r-base
version     : 3.5.1
build       : hea2b9be_1007
build number: 1007
size        : 33.9 MB
license     : GPL-3.0
subdir      : linux-64
url         : https://conda.anaconda.org/conda-forge/linux-64/r-base-3.5.1-hea2b9be_1007.tar.bz2
md5         : ac92f526a44386300d11a159dff827b8
timestamp   : 2019-06-18 20:39:58 UTC
dependencies: 
  - _r-mutex 1.* anacondar_1
  - bwidget
  - bzip2 >=1.0.6,<2.0a0
  - cairo >=1.16.0,<1.17.0a0
  - curl
  - freetype >=2.9.1,<3.0a0
  - gcc_linux-64 7.*
  - gfortran_linux-64 7.*
  - glib >=2.58.3,<3.0a0
  - gsl >=2.5,<2.6.0a0
  - gxx_linux-64 7.*
  - icu >=58.2,<59.0a0
  - jpeg >=9c,<10a
  - krb5 >=1.16.3,<1.17.0a0
  - libcurl >=7.64.1,<8.0a0
  - libgcc-ng >=7.3.0
  - libgfortran-ng >=7,<8.0a0
  - libpng >=1.6.35,<1.7.0a0
  - libssh2 >=1.8.2,<1.9.0a0
  - libstdcxx-ng >=7.3.0
  - libtiff >=4.0.9,<5.0a0
  - libuuid >=2.32.1,<3.0a0
  - libxml2 >=2.9.9,<2.10.0a0
  - make
  - pango >=1.40.14,<1.41.0a0
  - pcre >=8.41,<9.0a0
  - readline >=7.0,<8.0a0
  - tk >=8.6.9,<8.7.0a0
  - tktable
  - xz >=5.2.4,<5.3.0a0
  - zlib >=1.2.11,<1.3.0a0


r-base 3.5.1 hfb2a302_1009
--------------------------
file name   : r-base-3.5.1-hfb2a302_1009.tar.bz2
name        : r-base
version     : 3.5.1
build       : hfb2a302_1009
build number: 1009
size        : 39.1 MB
license     : GPL-3.0
subdir      : linux-64
url         : https://conda.anaconda.org/conda-forge/linux-64/r-base-3.5.1-hfb2a302_1009.tar.bz2
md5         : bab249168b1d0ce009d70d2c206c6ec8
timestamp   : 2019-07-24 16:55:39 UTC
dependencies: 
  - _r-mutex 1.* anacondar_1
  - bwidget
  - bzip2 >=1.0.8,<2.0a0
  - cairo >=1.16.0,<1.17.0a0
  - curl
  - freetype >=2.9.1,<3.0a0
  - gcc_linux-64 7.*
  - gfortran_linux-64 7.*
  - glib >=2.58.3,<3.0a0
  - gsl >=2.5,<2.6.0a0
  - gxx_linux-64 7.*
  - icu >=64.2,<65.0a0
  - jpeg >=9c,<10a
  - krb5 >=1.16.3,<1.17.0a0
  - libcurl >=7.64.1,<8.0a0
  - libgcc-ng >=7.3.0
  - libgfortran-ng >=7,<8.0a0
  - libpng >=1.6.35,<1.7.0a0
  - libssh2 >=1.8.2,<1.9.0a0
  - libstdcxx-ng >=7.3.0
  - libtiff >=4.0.10,<5.0a0
  - libuuid >=2.32.1,<3.0a0
  - libxml2 >=2.9.9,<2.10.0a0
  - make
  - pango >=1.42.4,<1.43.0a0
  - pcre >=8.41,<9.0a0
  - readline >=8.0,<9.0a0
  - tk >=8.6.9,<8.7.0a0
  - tktable
  - xz >=5.2.4,<5.3.0a0
  - zlib >=1.2.11,<1.3.0a0

```

**Comment by hermidalc** [comment]
I see in #16 that 3 days ago you in fact did a rebuild for icu 64.2 but the latest package at least for linux and r-base 3.5.1 still is depending on icu 58.*

**Comment by dbast** [comment]
Thanks for reporting. Also debugged this the last minutes... seems like during #16 there were either pinning issues or the r-base==3.5.1 rebuild for icu 64.2 was not yet available. A bump of the build number + rebuild should fix this.

Also tagging @uday1889 who also looked into this.

**Comment by hermidalc** [comment]
> Thanks for reporting. Also debugged this the last minutes... seems like during #16 there were either pinning issues or the r-base==3.5.1 rebuild for icu 64.2 was not yet available. A bump of the build number + rebuild should fix this.

I thought so too after reading #16 that there must've been some race condition between r-base and r-stringi releases

**Comment by dbast** [comment]
PR is open. Let's see what ICU versions are used there during the builds.

**Comment by dbast** [comment]
Already fixed via https://github.com/conda-forge/r-stringi-feedstock/pull/17

@hermidalc Can you try again?

**Comment by hermidalc** [comment]
> Already fixed via #17
> 
> @hermidalc Can you try again?

Looks good now, got 1009 build of r-base and new r-stringi that you built on icu 64.2.  Tested loading library too and no issues.  Thanks!

```bash
$ conda update --all
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/hermidalc/soft/miniconda3/envs/ml-bio-sklearn


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    cairo-1.16.0               |    hfb77d84_1002         1.5 MB  conda-forge
    fontconfig-2.13.1          |    h86ecdb6_1001         340 KB  conda-forge
    fribidi-1.0.5              |    h516909a_1002         112 KB  conda-forge
    harfbuzz-2.4.0             |       h9f30f68_2         1.5 MB  conda-forge
    libxslt-1.1.32             |    h31b3aaa_1004         520 KB  conda-forge
    matplotlib-3.1.1           |           py37_1           6 KB  conda-forge
    matplotlib-base-3.1.1      |   py37he7580a8_1         6.7 MB  conda-forge
    pango-1.42.4               |       he7ab937_0         515 KB  conda-forge
    qt-5.9.7                   |       h0c104cb_3        80.8 MB  conda-forge
    r-base-3.5.1               |    hfb2a302_1009        39.1 MB  conda-forge
    r-stringi-1.4.3            |    r35h0e574ca_3         767 KB  conda-forge
    rpy2-2.9.4                 |py37r351h42aa8d9_2         292 KB  conda-forge
    simplegeneric-0.8.1        |             py_1           7 KB  conda-forge
    ------------------------------------------------------------
                                           Total:       132.0 MB

The following NEW packages will be INSTALLED:

  fribidi            conda-forge/linux-64::fribidi-1.0.5-h516909a_1002
  simplegeneric      conda-forge/noarch::simplegeneric-0.8.1-py_1

The following packages will be UPDATED:

  cairo                                1.16.0-h18b612c_1001 --> 1.16.0-hfb77d84_1002
  fontconfig                           2.13.1-he4413a7_1000 --> 2.13.1-h86ecdb6_1001
  harfbuzz                                 2.4.0-h37c48d4_1 --> 2.4.0-h9f30f68_2
  icu                                    58.2-hf484d3e_1000 --> 64.2-he1b5a44_0
  libxslt                              1.1.32-hae48121_1003 --> 1.1.32-h31b3aaa_1004
  matplotlib                                   3.1.1-py37_0 --> 3.1.1-py37_1
  matplotlib-base                      3.1.1-py37hfd891ef_0 --> 3.1.1-py37he7580a8_1
  pango                               1.40.14-he7ab937_1005 --> 1.42.4-he7ab937_0
  python                                   3.7.3-h5b0a415_0 --> 3.7.3-h33d41f4_1
  qt                                       5.9.7-h52cfd70_2 --> 5.9.7-h0c104cb_3
  r-base                                3.5.1-hea2b9be_1007 --> 3.5.1-hfb2a302_1009
  r-stringi                             1.4.3-r35h0357c0b_2 --> 1.4.3-r35h0e574ca_3
  readline                                7.0-hf8c457e_1001 --> 8.0-hf8c457e_0
  rpy2                             2.9.4-py37r351h941a26a_1 --> 2.9.4-py37r351h42aa8d9_2
  sqlite                                  3.28.0-h8b20d00_0 --> 3.29.0-hcee41ef_0

The following packages will be DOWNGRADED:

  libxml2                                  2.9.9-h13577e0_2 --> 2.9.9-hee79883_2
```
